### DISCUSSION TITLE
블로그는 사드세요... 제발 프롤로그: 블로그를 만들자 | 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#15] [Announcements]
# 블로그는 사드세요... 제발 프롤로그: 블로그를 만들자 | 정현수 기술 블로그

블로그를 만드는 이유와 개발환경에 대해서

https://junghyeonsu-dev.vercel.app/posts/please-buy-the-blog-prolog

**Comment by deli-ght** [comment]
쉽지 않은 개인 사업자의 삶...

**Reply by junghyeonsu** [reply]
ㅋㅋㅋ... CTO 해주세요...

**Comment by o-bard-o** [comment]
이 글을 읽고 블로그를 해먹어 보고 싶어졌어요.

**Reply by junghyeonsu** [reply]
저만 따라오세요.

**Comment by BackdevHong** [comment]
홀리몰리..

**Comment by solidw** [comment]
저도 요즘 블로그를 만들고 있어서 포스팅 챙겨보고 있습니다 ㅎㅎ 꾸준히 연재해주세요~~

\+ 개인적으로 메인 화면에서 접근성도 고려되면 좋겠다는 생각이 들었습니다. 각 article에 role=button 같은 속성이 있으면 SEO에 더 도움이 되지 않을까 해요~

**Reply by junghyeonsu** [reply]
감사합니다~~~ article에 role=button은 소중하네요.. 꼭 적용해보도록 하겠습니다!
